src/quo2/components/wallet/account_overview/view.cljs

```diff
@@ -0,0 +1,148 @@
+            [quo2.theme :as quo.theme]))
+
+(defn- loading-state
+  [color]
+  [:<>
+   [rn/view
+    (style/loading-bar-margin-bottom {:color  color
+                                      :width  104
+                                      :height 14})]
+   [rn/view
+    (style/loading-bar-margin-bottom {:color  color
+                                      :width  152
+                                      :height 20})]
+   [rn/view {:style {:flex-direction :row}}
+    [rn/view {:style (style/loading-bar-margin-right color)}]
+    [rn/view {:style (style/loading-bar-margin-right color)}]
+    [rn/view
+     {:style (style/loading-bar-margin-right-extra {:color        color
+                                                    :width        62
+                                                    :height       10
+                                                    :margin-right 4})}]
+    [rn/view {:style (style/loading-bar-tiny color)}]]])
+
+(defn- time-string
+  [time-frame time-frame-string]
+  (case time-frame
+    :one-week     (i18n/label :t/one-week-int)
+    :one-month    (i18n/label :t/one-month-int)
+    :three-months (i18n/label :t/three-months-int)
+    :one-year     (i18n/label :t/one-year)
+    :all-time     (i18n/label :t/all-time)
+    time-frame-string))
+
+(defn- custom-time-frame
+  [time-frame-to-string]
+  [:<>
+   [icons/icon :i/positive-right
+    {:size            16
+     :container-style style/right-arrow}]
+   [quo2/text
+    {:weight :medium
+     :size   :paragraph-2
+     :style  style/bottom-time-to-text}
+    time-frame-to-string]])
+
+(defn- numeric-changes
+  [percentage-change currency-change customization-color theme up?]
+  [:<>
+   [quo2/text
+    {:weight :medium
+     :size   :paragraph-2
+     :style  (style/percentage-change (colors/theme-colors
+                                       (colors/custom-color customization-color 50)
+                                       (colors/custom-color customization-color 60)
+                                       theme))}
+    percentage-change]
+   [rn/view
+    {:style (style/dot-separator (colors/theme-colors
+                                  (colors/custom-color customization-color 60 40)
+                                  (colors/custom-color customization-color 50 40)
+                                  theme))}]
+   [quo2/text
+    {:weight :medium
+     :size   :paragraph-2
+     :style  (style/currency-change (colors/theme-colors
+                                     (colors/custom-color customization-color 50)
+                                     (colors/custom-color customization-color 60)
+                                     theme))}
+    currency-change]
+   [icons/icon
+    (if up?
+      :i/positive
+      :i/negative)
+    {:color (colors/theme-colors (colors/custom-color customization-color 50)
+                                 (colors/custom-color customization-color 60)
+                                 theme)
+     :size  16}]])
```

Comment:
I think these `colors/custom-color` should be called from the style file. Also they are repeated 4 times -- they can be extracted into one definition